Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Cancel/DefaultCancelButtonFormStyle.swift

```diff
@@ -1,6 +1,7 @@
 import UIKit
 
 public struct DefaultCancelButtonFormStyle: ElementButtonStyle {
+    public var textAlignment: NSTextAlignment = .natural
```

Comment:
I've tried to do a search in the PR for `textAlignment` and its been used 82 times. I can't see from the task or the code changes why that is needed, especially to the extent its been done. Do you think something that repetitive may be worth its own task, or is there any way we can understand why it was needed in the first place?

Reply:
_why it was needed in the first place?_ it's text alignment :)

Reply:
Well yes, I get that, but moving all of `UITextInput` properties into our own version of UI declaration doesn't really help. I think we went overboard with properties already (see height, width, isHidden, isEnabled, etc.

This is why before we add anything new I think we should really evaluate whether its needed and what for, can we do without, can we strip something useless and replace... We can't just keep migrating UIControl, UITextInput and UIView through our configurations...

Reply:
We can discuss required changes for these Style properties with team

Reply:
Absolutely. As part of the PR though I would still like to understand what problem you've tried to solve that required you to add this extra property, that imo we should scope separately?